user_modules/utils.lua

```diff
@@ -0,0 +1,84 @@
+  local keyset = {"year", "month", "day", "hour", "min", "sec"}
+  local count = 1
+  for element in string.gmatch(tz_date,'%d+') do
+    tz_table[keyset[count]] = element
+    count = count + 1
+  end
+  return os.time(tz_table)
+end
+
+--! @brief Allows to get struct value from any mobile api struct
+--! @param struct_name - name of needed struct
+--! @param param_name - struct parameter
+--! @param value_to_read - value which is needed to be read
+--! @usage Function usage example: maxvalueMenuParams = module.GetStructValueFromMobileApi( "MenuParams", "parentID", "maxvalue")
+function module.GetStructValueFromMobileApi(struct_name, param_name, value_to_read)
+  if not interface_schema.struct[struct_name] then
+    common_functions:userPrint(31, "Struct with name: ".. struct_name.. " does not exist")
+    return nil
+  end
+  if not interface_schema.struct[struct_name].param[param_name] then
+    common_functions:userPrint(31, "Param with name: ".. param_name.." does not exist in structure: "..struct_name )
+    return nil
+  end
+  return interface_schema.struct[struct_name].param[param_name][value_to_read]
+end
+
+--! @brief Function allows to get any enum size(number of elements) from mobile api
+--! @param enum_name - enum name which size we are going to get
+--! @param Function usage example: maxlength = enum_size = module.GetEnumSizeFromMobileApi("AppInterfaceUnregisteredReason")
+function module.GetEnumSizeFromMobileApi(enum_name)
+  if not interface_schema.enum[enum_name] then
+    common_functions:userPrint(31, "Enum with name: ".. enum_name.. " does not exist")
+    return nil
+  end
+  return #module.GetUnorderedTableKeyset(interface_schema.enum[enum_name])
+end
+
+--! @brief Function allows to get value from any mobile api function
+--! @param function_type - request, response or notification
+--! @param function_name - name of the function
+--! @param param_name - function parameter
+--! @param value_to_read - value which is needed to be read
+--! @param Function usage example: maxlength = module.GetFunctionValueFromMobileApi("request", "Show", "mainField2", "maxlength")
+function module.GetFunctionValueFromMobileApi(function_type, function_name, param_name, value_to_read)
+  if not interface_schema.type[function_type] then
+    common_functions:userPrint(31, "Function with type: ".. function_type.." does not exist")
+    return nil
+  end
+  if not interface_schema.type[function_type].functions[function_name] then
+    common_functions:userPrint(31, "Function with name: ".. function_name.. " does not exist")
+    return nil
+  end
+  if not interface_schema.type[function_type].functions[function_name].param[param_name] then
+    common_functions:userPrint(31, "Parameter with name: "..param_name.. " does not exist")
+    return nil
+  end
+  return interface_schema.type[function_type].functions[function_name].param[param_name][value_to_read]
+end
+
+return module
```

Comment:
@dboltovskyi Functions are related to loading API will not be too many for splitting them in new file. In current situation when a lot of functions are in many files it would be not usable if 2 files will be created for almost the same cause.